cogniac/cogniac.py

```diff
@@ -438,6 +439,23 @@ def get_camera(self, network_camera_id):
         """
         return CogniacNetworkCamera.get(self, network_camera_id)
 
+    def get_all_gateways(self):
```

Comment:
edgeflows instead of gateways?

Reply:
Yeah, that'd be better. I'll make those changes.